tasks/service.js

```diff
@@ -1,196 +1,169 @@
-// Generated by CoffeeScript 1.9.1
+'use strict';
 
-/*
-(c) js.seth.h@gmail.com
-MIT License
- */
+// Based on code (c) js.seth.h@gmail.com, MIT License
 
-(function() {
-  var existProcess, fs, log, loopUntil;
+const fs = require('fs');
+const child_process = require('child_process');
 
-  fs = require('fs');
+function existProcess(pid) {
+	try {
+		process.kill(pid, 0);
+		return true;
+	} catch (e) {}
+	return false;
+};
 
-  log = void 0;
+function loopUntil(fnEndOk, done) {
+	let loopId;
+	return loopId = setInterval(function() {
+		if (true === fnEndOk()) {
+			clearInterval(loopId);
+			return done();
+		}
+	}, 200);
+};
 
-  existProcess = function(pid) {
-    var err;
-    try {
-      process.kill(pid, 0);
-    } catch (_error) {
-      err = _error;
-      return false;
-    }
-    return true;
-  };
+module.exports = function(grunt) {
+	const log = grunt.log;
```

Comment:
It probably can just use `grunt.log` like everything else does.  It's just because the code came from a third party with a different style.